app/utils/airtable_fields_config.py

```diff
@@ -159,7 +159,6 @@
     'JBI 7': 'JBI_7',
     'JBI 8': 'JBI_8',
     'JBI 9': 'JBI_9',
-    'Superceded?': 'SUPERCEDED',
```

Comment:
Can get rid of this side of things now - we don't need the old variable names as we don't use them anywhere on research, now fully deprecated

Reply:
i think simona got rid of this in a sep PR already. I just build my branch off of it so the changes were reflected in the diff when her changes hadn't been merged in yet